pkg/apis/extensions/v1alpha1/types_worker.go

```diff
@@ -125,10 +125,10 @@ type WorkerPool struct {
 	Minimum int32 `json:"minimum"`
 	// Name is the name of this worker pool.
 	Name string `json:"name"`
-	// OSCHash is a hash of selected aspects of the OperatingSystemConfig. If the hash changes, then the
+	// NodeAgentSecretName is a hash of selected aspects of the OperatingSystemConfig. If the hash changes, then the
```

Comment:
I find the description confusing. `NodeAgentSecretName is a hash` raises more questions than it answers. Documentation should clarify and not confuse. Please adapt.

Maybe you can state something along the lines of "NodeAgentSecretName is uniquely identifying selected aspects of the OperatingSystemConfig. If it changes, ...".

Reply:
Changed. Thanks.